docs/apm/troubleshooting.asciidoc

```diff
@@ -0,0 +1,33 @@
+[[troubleshooting]] 
+=== Troubleshooting common problems
+
+If you have something to add to this section, please consider creating a pull request with
+your proposed changes at https://github.com/elastic/kibana.
+
+Also check out the https://discuss.elastic.co/c/apm[APM discussion forum].
+
+==== No APM data found
+
+This section can help with any of the following:
+
+* Data isn't displaying in the APM app
+* You're seeing a message like "No Services Found",
+* You're seeing errors like "Fielddata is disabled on text fields by default..."
+
+There are a number of factors that could be at play here.
+One important thing to double-check first is your index template.
+
+An APM Index Template must exist for the APM app to work correctly.
+By default, this index template is created by APM Server on startup.
+However, this only happens if `setup.template.enabled` is `true` in `apm-server.yml`.
+You can create the index template manually by running `apm-server setup`.
+Take note that index templates *cannot* be applied retroactively -- they are only applied at index creation time.
+More information is available in {apm-server-ref}/apm-server-configuration.html[Set up and configure].
+
+If you're not outputting data from APM Server directly to Elasticsearch, you'll need to
+{apm-server-ref}/_manually_loading_template_configuration.html[load the template manually].
```

Comment:
@sqren, that's what I was attempting to say here. I'll make it more clear and update the link to point to that anchor. Thanks!